webpack.config.js

```diff
@@ -329,7 +330,7 @@ const webpackConfig = {
 					chunkGroups: true,
 				},
 			} ),
-		shouldEmitStats && new webpack.ProgressPlugin( createProgressHandler() ),
+		shouldShowProgress && new webpack.ProgressPlugin( createProgressHandler() ),
```

Comment:
Here's an opportunity to do something we should do a long time ago: move `createProgressHandler` to its own module, so that the main webpack config is less cluttered.

Reply:
moved so calypso-build!